### PR TITLE
remove query interface

### PR DESCRIPTION
**Pull request by zakird**

### CODE REVIEW

File: censys/__init__.py [hunk 1]
```diff
@@ -1,7 +1,8 @@
 __author__ = "Censys Team"
-__email__ = "team@censys.io"
-__copyright__ = "Copyright 2015 Regents of the University of Michigan"
-__version__ = "0.0.8"
+__email__ = "support@censys.io"
+__copyright__ = "Copyright 2017 ZCorp Technology"
+__version__ = "0.0.10"
```
Comment (dadrian) [inline censys/__init__.py]: Shouldn't this be 0.1.0, since we're making a breaking API change?

File: censys/ipv4.py [hunk 2]
```diff
@@ -41,9 +41,14 @@ def testBadFieldsSearch(self):
...
+            "location.country", 100))
 
 
 if __name__ == "__main__":
-    unittest.main()
+    #unittest.main()
+
+    api = CensysIPv4()
+    for r in api.search(r'("var BLANK_URL" OR "Mage.Cookies" OR "js\/varien\/js.js" OR "\/skin\/frontend\/") AND 80.http.get.status_code:200 AND "\\x6F\\x6E\\x65\\x70\\x61\\x67\\x65\\x7C\\x63\\x68\\x65\\x63\\x6B\\x6F\\x75\\x74"'):
+        print(r)
```
Comment (dadrian) [inline censys/ipv4.py]: What?

**Comment by zakird** [comment]
I swear that was just in master and broke in the merge, but it's clearly
terrible, so I'll fix regardless.

**Comment by zakird** [comment]
Both comments should be addressed.

**Pull request merged**